remote_server/remote_server_installer.sh

```diff
@@ -0,0 +1,512 @@
+            ln -s $INIT_FILE /etc/rc2.d/S99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/rc3.d/S99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/rc4.d/S99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/rc5.d/S99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/rc6.d/K99revealmetrics_remote_server
+        elif [ -d "/etc/init.d/rc1.d" ]; then
+            rm -f /etc/init.d/rc*.d/*revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/init.d/rc1.d/K99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/init.d/rc2.d/S99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/init.d/rc3.d/S99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/init.d/rc4.d/S99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/init.d/rc5.d/S99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/init.d/rc6.d/K99revealmetrics_remote_server
+            ln -s $INIT_FILE /etc/init.d/rcS.d/S99revealmetrics_remote_server
+        fi
+
+        echo
+        echo "Thank you for setting up the Uptime Cloud Monitor Metrics Agent."
+        echo "You may use 'sudo $INIT_FILE [start|stop]' to start/stop the agent"
+        echo
+        $INIT_FILE start
+        echo
+    fi
+}
+
+create_exe_file()
+{
+    LAUNCHER_FILE="/usr/local/copperegg/ucm-metrics/revealmetrics_remote_server_launcher.sh"
+    if [ -n "$RVM_SCRIPT" ]; then
+        cat <<ENDINIT > $LAUNCHER_FILE
+#!/bin/bash
+DIRNAME="`dirname \$0`"
+. $RVM_SCRIPT
+cd \$DIRNAME
+$AGENT_FILE \$*
+ENDINIT
+        EXE_FILE=$LAUNCHER_FILE
+    elif [ -n "$RUBY_PATH" ]; then
+        cat <<ENDINIT > $LAUNCHER_FILE
+#!/bin/sh
+PATH=$RUBY_PATH:\$PATH
+export PATH
+$AGENT_FILE \$*
+ENDINIT
+        EXE_FILE=$LAUNCHER_FILE
+    else
+        EXE_FILE=$AGENT_FILE
+    fi
+    chmod +x $EXE_FILE
+    echo $EXE_FILE
+}
+
+
+###############################################################
+###############################################################
+##   End Functions, start main code
+###############################################################
+###############################################################
+
+if [ -z "$FREQ" -o -z "`echo $FREQ | egrep -o '^[0-9]$'`" ]; then
+    # FREQ is an empty string or not a number.  Ask for it
+    echo
+    echo "Monitoring frequency (sample rate) can be one sample per: 60, 300, 900, 3600 seconds."
+    echo -n "What frequency would you like? [60] "
+    read FREQ
+    if [ -z "`echo $FREQ | egrep -o '^[0-9]+$'`" ]; then
+        # freq is not a number. default to 60
+        FREQ=60
+    elif [ $FREQ -le 60 ]; then
+        FREQ=60
+    elif [ $FREQ -le 300 ]; then
+        FREQ=300
+    elif [ $FREQ -le 900 ]; then
+        FREQ=900
+    else
+        FREQ=3600
+    fi
+fi
+echo "Monitoring frequency set to one sample per $FREQ seconds."
+
+echo
+
+echo "Installing required gems (using bundler)"
+gem install bundler -v1.10.6
```

Comment:
if it is installed, nothing would happen. I feel no point adding additional check. 
If another version is installed, let it install 1.10.6 so that gemfile.lock warning doesn't appear (`The bundler version differ in gemfile` warning)